packages/drivers/odsp-driver/src/test/odspDriverResolverTest.spec.ts

```diff
@@ -102,20 +102,6 @@ describe("Odsp Driver Resolver", () => {
         assert.strictEqual(resolvedUrl.shareLinkInfo.createLink.type, createLinkType);
     });
 
-    it("Should resolve url with a string in the codeDetails package", async () => {
```

Comment:
While there's still back compat code in odsp-driver, we should keep these tests to cover it.  Once that code is removed would be when we should remove these tests.